### ISSUE TITLE
Recording doesn't stop

### ISSUE THREAD

**Issue by fulopsoma23** [Michele0303/tiktok-live-recorder#190]
I noticed that if I record several hours of live, the program does not want to stop after the live is finished. It does not respond to CTRL+C either. Unfortunately, my recordings become corrupt when this happens.

**Comment by Michele0303** [comment]
Did you press only once or repeatedly?

**Comment by fulopsoma23** [comment]
> Did you press only once or repeatedly?

I pressed it only once, but after several minutes it still does not respond.

**Comment by Michele0303** [comment]
On what operating system and terminal?

**Comment by fulopsoma23** [comment]
> On what operating system and terminal?

I use Windows 11 and the factory cmd terminal.

**Comment by Michele0303** [comment]
Has this ever happened to you in other versions?

**Comment by fulopsoma23** [comment]
> Has this ever happened to you in other versions?

Yes, unfortunately I had problems with my previous system. I usually record live for several hours (sometimes 4-5 hours).

**Comment by Michele0303** [comment]
Has this ever happened to you using the '-ffmpeg' flag?

**Comment by fulopsoma23** [comment]
Yes, this also happened with the ffmpeg command. I tried to update it, but it didn't solve the problem.

**Comment by Michele0303** [comment]
Strange, it has never happened to me, I will try to record longer live to test better.

Does this happen to you often?

**Comment by fulopsoma23** [comment]
Yes, unfortunately, it almost always happens. If the live lasts 20-30 minutes, there is usually no problem with the recording.

**Comment by jotherthings** [comment]
I also regularly record 2-3 hours live, this is what I did, every couples minutes I check "date modified" on the file, If it's downloading, date modified should be the same as your clock. If my clock 11:23 but date modified on the file 11:20, I ctrl-c the program and start the program again.
The file most likely will be corrupted, I fixed the file with untruct app.
On previous version, I just select-unselect-select the file and the date modified should change, but right now if I want to know date modified of the file, I should close the folder-open the folder again-select file, and the date modified should change.
